x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/operator/OperatorOnlyRegistry.java

```diff
@@ -35,7 +36,8 @@
         // Repository analysis actions are not mentioned in core, literal strings are needed.
         "cluster:admin/repository/analyze",
         "cluster:admin/repository/analyze/blob",
-        "cluster:admin/repository/analyze/blob/read"
+        "cluster:admin/repository/analyze/blob/read",
+        ClientEnrollmentAction.NAME
```

Comment:
Just seen Albert's comment here for Node Enrollment action. May be applicabble here as well.

Reply:
Yeap, see my comment on your PR too. Will update